vendor: bump pebble to 382a403837cb2edfc5d60d4a4476e61c17ac435e

petermattis opened the pull request:
Picks up the fix to never compress the meta-index block which causes
badness and triggers assertions inside of RocksDB.

Fixes #43020

Release note: None

petermattis committed:
vendor: bump pebble to 382a403837cb2edfc5d60d4a4476e61c17ac435e

Picks up the fix to never compress the meta-index block which causes
badness and triggers assertions inside of RocksDB.

Fixes #43020

Release note: None

cockroach-teamcity commented:
<!-- Reviewable:start -->
This change is [<img src="https://reviewable.io/review_button.png" height="34" align="absmiddle" alt="Reviewable"/>](https://reviewable.io/reviews/cockroachdb/cockroach/43029)
<!-- Reviewable:end -->

itsbilal reviewed (approved):
<img class="emoji" title=":lgtm:" alt=":lgtm:" align="absmiddle" src="https://reviewable.io/lgtm.png" height="20" width="61"/>

> *[Reviewable](https://reviewable.io/reviews/cockroachdb/cockroach/43029#-:-LvRkJzV1RAVTdQg-X61:bnfp4nl)* status: :shipit: complete! 1 of 0 LGTMs obtained (waiting on @ajkr and @itsbilal)



<!-- Sent from Reviewable.io -->

petermattis commented:
TFTR! Now to wait 30min for CI, and another 30mi for bors.

petermattis commented:
Preemptively bors'ing as CI looks like it will finish soon.

bors r+

petermattis commented:
CI failed various `current_time` logic tests. Unrelated to this PR.

bors r+

petermattis committed:
sql/logictest: skip a portion of the time/current_time_tests

Disable a portion of the `time/current_time_tests` which is flaky under
CI and locally.

See #43036

Release note: None

knz commented:
Test skip LGTM.

knz commented:
Issuing bors because I need this in a separate CI run.

bors r=itsbilal

andreimatei commented on pkg/sql/logictest/testdata/logic_test/time:
```diff
@@ -444,26 +444,29 @@ SELECT id FROM current_time_test WHERE
-
-statement ok
-insert into current_time_tzset_test (id, a) values (1, current_time)
+# TODO(otan): this is flaky. Failure can be reproduced with:
```
please list the issue number

knz replied:
I'd rather not interrupt this build now.
Anyway I've referred to this skip as a comment in #43036.

petermattis replied:
I will do better next time.
